publish/src/main/java/com/indix/gocd/s3publish/PublishExecutor.java

```diff
@@ -179,43 +252,66 @@ private void setMetadata(GoEnvironment env, String bucket, String destinationPre
         store.put(putObjectRequest);
     }
 
-    private String getConfigDestinationPrefix(final TaskConfig config) {
+    private String getConfigDestinationPrefix( final TaskConfig config) {
```

Comment:
Please avoid stray changes. @ashwanthkumar @manojlds perhaps adding a style validator or having code formatting rules might help to reduce these?